igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/stash/client/model/AbstractStashResponse.groovy

```diff
@@ -20,7 +20,6 @@ abstract class AbstractStashResponse {
     int size
     int limit
     boolean isLastPage
-    List<?> values
```

Comment:
I'm also curious why you removed this

Reply:
Answer in the bitbucket class.

Reply:
I guess what I'm wondering is - is this being used elsewhere? will this break existing behavior? groovy doesn't always tell you if you remove a property that is in use.

Reply:
It was not. I checked. There was a single subclass using that field.